src/components/navigation/icons/EmptyState.tsx

```diff
@@ -1,104 +1,21 @@
-              opacity=".8"
-            />
-            <path
-              fill="#01ACCD"
-              d="M5.5 2.5L8 2.5 8 5.5 5.5 5.5 5.5 8 2.5 8 2.5 5.5 0 5.5 0 2.5 2.5 2.5 2.5 0 5.5 0z"
-              opacity=".3"
-              transform="translate(164 109)"
-            />
-            <path
-              fill="#01ACCD"
-              d="M66.302 157.649c1.823 0 3.301-1.489 3.301-3.325S68.125 151 66.302 151c-1.824 0-3.302 1.488-3.302 3.324 0 1.836 1.478 3.325 3.302 3.325zm0-2c-.716 0-1.302-.59-1.302-1.325 0-.734.586-1.324 1.302-1.324.715 0 1.301.59 1.301 1.324 0 .735-.586 1.325-1.301 1.325z"
-              opacity=".3"
-            />
-            <path
-              fill="#01ACCD"
-              d="M5.5 2.5L8 2.5 8 5.5 5.5 5.5 5.5 8 2.5 8 2.5 5.5 0 5.5 0 2.5 2.5 2.5 2.5 0 5.5 0z"
-              opacity=".3"
-              transform="translate(12 76)"
-            />
-            <g fill="#01ACCD" opacity=".9">
-              <path d="M5.5 2.5L8 2.5 8 5.5 5.5 5.5 5.5 8 2.5 8 2.5 5.5 0 5.5 0 2.5 2.5 2.5 2.5 0 5.5 0z" transform="translate(71 164)" />
-            </g>
-            <g fill="#01ACCD" opacity=".8">
-              <path
-                d="M3.302 6.649c1.823 0 3.301-1.489 3.301-3.325S5.125 0 3.302 0C1.478 0 0 1.488 0 3.324 0 5.16 1.478 6.65 3.302 6.65zm0-2C2.586 4.649 2 4.059 2 3.324 2 2.59 2.586 2 3.302 2c.715 0 1.301.59 1.301 1.324 0 .735-.586 1.325-1.301 1.325z"
-                transform="translate(13 102)"
-              />
-            </g>
-          </g>
-        </g>
-      </svg>
-    </div>
+      <div class={styles.emptyState}>
+        <div class={styles.arrow}>
+          <svg width="109" height="51" viewBox="0 0 109 51" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M104.114 32.428C107.014 36.4279 108.747 42.1111 108.999 49.9841C109.008 50.2601 108.791 50.491 108.515 50.4998C108.239 50.5086 108.008 50.292 108 50.016C107.751 42.264 106.048 36.7991 103.305 33.0148C100.569 29.2402 96.76 27.0851 92.1896 26.0218C87.6073 24.9558 82.2725 24.9912 76.5247 25.5991C70.7792 26.2068 64.6581 27.3819 58.5056 28.575L58.0569 28.662C52.0616 29.8249 46.0449 30.9919 40.3676 31.6399C34.5506 32.3039 29.0528 32.428 24.2647 31.43C19.466 30.4298 15.362 28.2979 12.3758 24.444C9.39539 20.5976 7.57109 15.0907 7.2174 7.4194C7.20468 7.14355 7.41798 6.90962 7.69384 6.8969C7.96968 6.88418 8.20361 7.09749 8.21633 7.37334C8.56417 14.9176 10.353 20.2007 13.1662 23.8315C15.9736 27.4546 19.8437 29.487 24.4687 30.451C29.1043 31.4172 34.4795 31.3055 40.2542 30.6464C45.8921 30.0029 51.8754 28.8424 57.8847 27.6768L58.3152 27.5933C64.4601 26.4017 70.6238 25.2177 76.4195 24.6047C82.2129 23.9919 87.6756 23.945 92.4162 25.0478C97.1686 26.1535 101.208 28.4185 104.114 32.428Z" fill="#CCCCCC"/>
```

Comment:
try to move the values into a const file and import from it.

Reply:
@semarche-kaltura please take a look at my latest PR

Reply:
changes looks good